extensions/kotlin.md

```diff
@@ -0,0 +1,158 @@
+# Kotlin
+
+[Kotlin](https://kotlinlang.org/) is a programming language which interoperates fully with Java and the JVM. As Axon is written in Java it can be used in conjunction with Kotlin too, offering a different feel to using the framework.
```

Comment:
```suggestion
[Kotlin](https://kotlinlang.org/) is a programming language which interoperates fully with Java and the JVM. As Axon is written in Java it can be used in conjunction with Kotlin too, offering a different feel when using the framework.
```